runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -106,9 +106,8 @@ object GrpcClientSettings {
       case "static" | "grpc-dns" =>
         val host = clientConfiguration.getString("host")
         require(host.nonEmpty, "host can't be empty when service-discovery-mechanism is set to static or grpc-dns")
-        // Required by the Discovery infrastructure, even when we use static discovery.
-        if (serviceName.isEmpty)
-          serviceName = "static"
+        // Required by the Discovery infrastructure, set to host as we use static or grpc-dns discovery.
+        serviceName = host
```

Comment:
should we make this `host:port` when the port is non-standard, or does netty take care of that for us?

Reply:
checking...

Reply:
I have no clue, sorry this is deeper than I can dig right now

Reply:
:+1: - this is already a good improvement, thanks!